updates/config.go

```diff
@@ -50,15 +50,21 @@ func registerConfig() error {
 				Value:       helper.ReleaseChannelBeta,
 			},
 			{
-				Name:        "Special",
-				Description: "Special releases or version changes for troubleshooting. Only use temporarily and when instructed.",
-				Value:       helper.ReleaseChannelSpecial,
+				Name:        "Support",
+				Description: "Support releases or version changes for troubleshooting. Only use temporarily and when instructed.",
+				Value:       helper.ReleaseChannelSupport,
 			},
 			{
 				Name:        "Staging",
 				Description: "Dangerous development releases for testing random things and experimenting. Only use temporarily and when instructed.",
 				Value:       helper.ReleaseChannelStaging,
 			},
+			// TODO: Remove as soon as everyone has switched away.
+			{
+				Name:        "Special (Deprecated!)",
+				Description: "This channel has been deprecated. If selected, the Stable channel will be used instead.",
```

Comment:
Who does guarantee that? Or ist stable Just a fallback for invalid values

Reply:
Yes, stable is fallback.